securedrop/source_templates/login.html

```diff
@@ -24,9 +24,9 @@ <h2>{{ gettext('Enter Codename') }}</h2>
   </div>
 
   <div class="form-controls">
-    <a href="{{ url_for('main.index') }}" class="btn secondary" aria-label="{{ gettext('Cancel') }}">
+    <a href="{{ url_for('main.index') }}" class="btn secondary" aria-label="{{ gettext('Cancel and Return to Previous Page') }}">
       {{ gettext('CANCEL') }}</a>
-    <button type="submit" aria-label="{{ gettext('Continue') }}">
+    <button type="submit" aria-label="{{ gettext('Continue to Submissions Page') }}">
```

Comment:
Nit here too, per <https://docs.securedrop.org/en/stable/source.html#making-your-first-submission>:

```suggestion
    <button type="submit" aria-label="{{ gettext('Continue to Submission Page') }}">
```